okhttp/src/commonMain/kotlin/okhttp3/internal/-HeadersCommon.kt

```diff
@@ -0,0 +1,192 @@
+/*
+ * Copyright (C) 2022 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package okhttp3.internal
+
+import okhttp3.Headers
+
+// TODO find out why [] doesn't fail and returns undefined
```

Comment:
I think it’s just how JavaScript arrays work

Reply:
So should I leave it with the existing behaviour?

Reply:
Your original behavior that defended against undefined is my preference.